extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -104,7 +104,10 @@ class ReplicationStatusProcessor {
             queueProcessor: this.processKafkaEntry.bind(this),
             bootstrap: options && options.bootstrap,
         });
-        this._consumer.on('error', () => {});
+        this._consumer.on('error', () => {
+            this.logger.fatal('error starting a backbeat consumer');
```

Comment:
I'm concerned that we would exit the process also after it has correctly connected to Kafka, in case errors are raised. Not that it's not necessarily a bad thing, since those processes should be stateless, but kafka client should handle reconnection automatically in such case, so we may want to refine e.g. by only exiting if `!this._consumerReady`.